packages/vuetify/src/components/VGrid/breakpoints.ts

```diff
@@ -0,0 +1,3 @@
+export const breakpoints = ['sm', 'md', 'lg', 'xl', 'xxl'] as const // no xs
```

Comment:
we need to come up with a way to propagate this sizing information as part of the framework simply because of how many places it is used in.

Reply:
After thinking on this, I felt the most natural place would be within the display composable. What do you think @vuetifyjs/core-team ?

Reply:
What about this change in display composable?

![image](https://user-images.githubusercontent.com/5698884/224214097-edf5cc3e-9ced-4b9d-95aa-a569dcc3aa1a.png)